terraform/aws/modules/genesis-support/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "share_name_test" {}
```

Comment:
just a preference, it's to have a descriptions in variables so we can get in a first glance the need of them. also it's nice each module to have it's own document. There is a nice tool which generates everything in markdown and leverages the description of variables and default values too https://github.com/terraform-docs/terraform-docs

Reply:
@spirosoik Yes I agree on that. We keep all the details in the main templates and not the actual modules but we need to start giving some love on the open source modules. I will start by adding description to this module here and later we can see about the document thing.